### PR TITLE
Add Q3 transparency report

### PR DESCRIPTION
**Pull request by lehnberg**
* Update grin-wallet-txs log
* Add Q3 transparency report
* Fix a typo in decision log

### CODE REVIEW

**Review by quentinlesceller — APPROVED** [review]
Clear and concise. Great job 👍

File: financials/reports/funding_transparency_2020Q3.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Status after the period
+
+### Disposable funds
+
+At the end of Sep 30 2020, Grin held disposable funds to the equivalent of $898,220.37 in the following assets:
+
+Currency | Amount | USD x-rate Sep 30 2020 | USD Equivalent | Wallet address(es)
+|---|---:|---:|---:|---|
+GRIN | 3770.71555489 | $0.306694 | $1,156.46 | https://donations.grin-tech.org
+BTC | 104.54381179 | $10,784.49 | $1,127,451.69 | 9hgKkhfPFpwKRQ6yARGwU2uuVay69NKu4<br />bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx
```
Comment (quentinlesceller) [inline financials/reports/funding_transparency_2020Q3.md:64]: I think the total was 104.514006 BTC here.

Comment (quentinlesceller) [reply financials/reports/funding_transparency_2020Q3.md:64]: Confirmed with BlockCypher here https://api.blockcypher.com/v1/btc/main/addrs/bc1qecw954uappnwhle6tmafqskn6vxmpr6lmmuwfwldphfxplcshgnssy9kmx see `ref_balance`

**Comment by quentinlesceller** [comment]
What's the status of this?

**Comment by lehnberg** [comment]
@quentinlesceller only getting to this now. I _think_ the discrepancy is a result from us not accounting for transaction fees. Do you agree? 

If so - We probably need to rethink how we're doing our bookkeeping.

File: financials/reports/funding_transparency_2020Q3.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+| | | | $1,128,608.15 |
+
+### Comments
+* Quarter over Quarter, the disposable funds increased in dollar terms by ~$230,000, attributed both to the BTC 6.42 net increase to the fund and positive movement in the BTC/USD exchange rate during the period.
+* The sole income received in the period was a 10 BTC donation from the Nervos foundation.
```
Comment (gringineer) [inline financials/reports/funding_transparency_2020Q3.md:69]: [Common Knowledge Base](https://github.com/nervosnetwork/ckb)

**Comment by tromp** [comment]
Do we know how much we paid in tx fees for each payout?

**Comment by quentinlesceller** [comment]
Will check

**Comment by lehnberg** [comment]
Closing, in favour of https://github.com/mimblewimble/grin-pm/pull/402

**Pull request closed without merging**